OrbitGl/ModulesDataView.cpp

```diff
@@ -163,22 +158,20 @@ void ModulesDataView::OnContextMenu(const std::string& action, int menu_index,
   }
 }
 
-//-----------------------------------------------------------------------------
-void ModulesDataView::OnTimer() {}
-
 //-----------------------------------------------------------------------------
 void ModulesDataView::DoFilter() {
   std::vector<uint32_t> indices;
   std::vector<std::string> tokens = absl::StrSplit(ToLower(m_Filter), ' ');
 
   for (size_t i = 0; i < modules_.size(); ++i) {
-    std::shared_ptr<Module>& module = modules_[i];
-    std::string name = ToLower(module->GetPrettyName());
+    const ModuleData* module = modules_[i];
+    std::string module_string = absl::StrFormat(
+        "%s %s", module->address_range(), ToLower(module->path()));
 
     bool match = true;
 
     for (std::string& filterToken : tokens) {
```

Comment:
nit: while you are at it: 
```suggestion
    for (std::string& filter_token : tokens) {
```